examples/src/main/scala/TracingExample.scala

```diff
@@ -54,7 +73,7 @@ object TracingExample extends IOApp.Simple {
             .build()
             .getOpenTelemetrySdk
         )
-      )(sdk => IO.unit /* TODO in 1.23, call shutdown() */ )
+      )(sdk => asyncFromCompletableResultCode(IO(sdk.getSdkTracerProvider.shutdown())))
```

Comment:
This is incomplete for the SDK as a whole, but it's necessary to get the example to work.  The missing method on the SDK is on main, but not released yet.